include/dataformats/GeoID.hpp

```diff
@@ -10,71 +10,87 @@
 #define DATAFORMATS_INCLUDE_DATAFORMATS_GEOID_HPP_
 
 #include <cstdint>
+#include <istream>
 #include <limits>
 #include <ostream>
-#include <istream>
 #include <tuple>
 
 namespace dunedaq {
 namespace dataformats {
 
+enum class GeoIDComponentType : uint16_t
+{
+  kTPC = 1,
+  kPDS = 2,
+  kDataSelection = 3,
+  kInvalid = 0
```

Comment:
I know it's annoying, but can we have the usual "ToString" method? And viceversa. In the TRBuilder we will need to configure this and I believe it will be useful. Maybe even in the operator <<.

Reply:
Done. Please check commit a31f3c7